frontend/src/index.html

```diff
@@ -86,10 +86,10 @@ <h4 class="text-sm font-semibold uppercase md:text-base">Audio Recordings</h4>
     <div class="mt-4 grid grid-cols-1 gap-2 md:grid-cols-2">
       {% for audio in audios offset: 25 limit: 8 %}
         <a href="./audios/{{ audio.objectID }}/">
-          <div class="grid  grid-cols-6 rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50 md:grid-cols-8">
-            <div class="col-span-1 ml-2">
+          <div class="flex items-center rounded-md border border-indigo-300 hover:!bg-yellow-100 hover:!bg-opacity-50">
+            <div class="mx-2 mt-1.5">
               <button
-                class="mt-2 w-8 flex-none rounded-full bg-neutral-600 fill-current focus:outline-none"
+                class="w-8 flex-none rounded-full bg-neutral-600 fill-current focus:outline-none"
```

Comment:
Here we have `svg` inside `button` inside `div`. I believe we can eliminate this hierarchy and use only `svg`. See the same block in search results. Simplify the markup.

Reply:
Not fully. I proposed getting rid of the `button` altogether. Keep only `svg` here.